ts/components/TosWebviewComponent.tsx

```diff
@@ -1,94 +1,90 @@
 import { NOTIFY_LINK_CLICK_SCRIPT } from "./ui/Markdown/script";
 import { WebViewMessage } from "./ui/Markdown/types";
-import { H2 } from "./core/typography/H2";
 
 type Props = {
   webViewSource: WebViewSource;
   handleLoadEnd: () => void;
   handleReload: () => void;
   onAcceptTos?: () => void;
-  onExit?: () => void;
   shouldRenderFooter?: boolean;
+  flow: FlowType;
 } & Pick<ViewProps, "testID">;
 
-const styles = StyleSheet.create({
-  errorContainer: {
-    padding: 20,
-    flex: 1,
-    justifyContent: "center",
-    alignItems: "center"
-  },
-  errorButtonsContainer: {
-    position: "absolute",
-    bottom: 30,
-    flex: 1,
-    flexDirection: "row"
-  }
-});
-
 const TosWebviewComponent: React.FunctionComponent<Props> = ({
   handleLoadEnd,
   handleReload,
   webViewSource,
   shouldRenderFooter,
-  onExit,
-  onAcceptTos
+  onAcceptTos,
+  flow
 }: Props) => {
   const [hasError, setHasError] = useState(false);
 
   const handleError = useCallback(() => {
     handleLoadEnd();
     setHasError(true);
-  }, [setHasError, handleLoadEnd]);
+    trackToSWebViewError(flow);
+  }, [handleLoadEnd, flow]);
 
   const handleRetry = useCallback(() => {
     handleReload();
     setHasError(false);
-  }, [setHasError, handleReload]);
+    trackToSWebViewErrorRetry(flow);
+  }, [handleReload, flow]);
 
   const renderError = () => (
-    <View style={styles.errorContainer} testID={"toSErrorContainerView"}>
-      <Image
-        source={brokenLinkImage}
-        resizeMode="contain"
-        testID={"toSErrorContainerImage"}
-      />
-      <View>
+    <>
+      <View
+        style={[
+          IOStyles.flex,
+          IOStyles.alignCenter,
+          IOStyles.horizontalContentPadding,
+          IOStyles.centerJustified
+        ]}
+        testID="toSErrorContainerView"
+      >
+        <Pictogram name="stopSecurity" />
         <VSpacer size={8} />
-        <H2 color="bluegrey" weight="Bold" testID={"toSErrorContainerTitle"}>
+        <H3 testID="toSErrorContainerTitle" style={{ textAlign: "center" }}>
           {I18n.t("onboarding.tos.error")}
-        </H2>
-      </View>
-
-      <View style={styles.errorButtonsContainer}>
-        <ButtonDefaultOpacity
-          onPress={handleRetry}
-          style={{ flex: 2 }}
-          block={true}
-          primary={true}
-          testID={"toSErrorContainerButton"}
-        >
-          <NBButtonText testID={"toSErrorContainerButtonText"}>
-            {I18n.t("global.buttons.retry")}
-          </NBButtonText>
-        </ButtonDefaultOpacity>
+        </H3>
       </View>
-    </View>
+      <FooterWithButtons
+        type="SingleButton"
+        sticky={true}
+        primary={{
+          type: "Solid",
+          buttonProps: {
+            onPress: () => handleRetry,
+            label: I18n.t("global.buttons.retry"),
+            accessibilityLabel: I18n.t("global.buttons.retry"),
+            testID: "RetryButtonTest"
+          }
+        }}
+      />
```

Comment:
I see no gradient in the screenshot of the new screen on the JIRA ticket. Am I right?

```suggestion
      <ContentWrapper>
          <ButtonSolid
            fullWidth
            onPress={handleRetry}
            label={retryButtonLabel}
            accessibilityLabel={retryButtonLabel}
            testID="RetryButtonTest"
          />
      </ContentWrapper>
```